### ISSUE TITLE
Rank field auto populate

### ISSUE THREAD

**Issue by evmallidis** [plantagoIT/strapi-drag-drop-content-type-plugin#27]
I would suggest that rank field should be auto populated. Now the user has to increment it right on new entry (if i'm not missing something). A solution would be for someone to update the entry aftercreate and put the identity(id) into rank field. Afterwards make the rank field disabled on view. That way would be populated and make it unique and ordered. That solution ofcourse is up to the developer and not the plugin, but maybe can help someone.

**Comment by derschiw** [comment]
Agree, might make sense to automatically add a `rank` for new items. I will take a look when I have time for this :)

**Comment by Ansis100** [comment]
Yes, auto populating ranks would reduce confusion when some rank fields are set, but some are not.

Also, that would make it possible for newly added items to be added to the end of the list, not the beginning, which is the current behaviour. It could be a settings toggle - add new items to the *beginning* or the *end* of the list.

**Comment by robozb** [comment]
would be necessary!!!
+1
